src/content/docs/kubernetes-pixie/k8s-otel/intro.mdx

```diff
@@ -0,0 +1,185 @@
+        <td>Telemetry data (HTTP)</td>
+        <td>Internal pod networks</td>
+        <td>Gathers telemetry data transmitted over HTTP protocol toward defined endpoints.</td>
+      </tr>
+      <tr>
+        <td>[Prometheus/ksm](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml) for `Deployment` configuration</td>
+        <td>Cluster state metrics</td>
+        <td>`kube-state-metrics` pods</td>
+        <td>Scrapes metrics relevant to Kubernetes resource states, such as deployments, pods, and nodes.</td>
+      </tr>
+      <tr>
+        <td>[Prometheus/controlplane](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml) for `Deployment` configuration</td>
+        <td>Control plane metrics</td>
+        <td>Kubernetes API server, scheduler, controller-manager</td>
+        <td>Monitors control plane components for operational metrics, performance, and availability statistics.</td>
+      </tr>
+      <tr>
+        <td>[K8s Events](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/templates/deployment-configmap.yaml) for `Deployment` configuration</td>
+        <td>Infrastructure events</td>
+        <td>Kubernetes Event API</td>
+        <td>Captures events like pod creations, deletions, scaling activities, and failure events.</td>
+      </tr>
+      <tr>
+        <td>[OTLP](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml) for `Deamonset` configuration</td>
+        <td>Telemetry data</td>
+        <td>Internal pod networks</td>
+        <td>Retrieves telemetry data from pods and containerized environments within the local network.</td>
+      </tr>
+      <tr>
+        <td>[Prometheus](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml) for `Deamonset` configuration</td>
+        <td>Node-level metrics</td>
+        <td>Host system metrics (e.g., node CPU, memory)</td>
+        <td>Collects metrics from host and node-level components to analyze resource consumption.</td>
+      </tr>
+      <tr>
+        <td>[K8s Events](https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/templates/daemonset-configmap.yaml) for `Deamonset` configuration</td>
+        <td>Infrastructure events</td>
+        <td>Kubernetes Event API</td>
+        <td>Similar to deployment, capturing Kubernetes events impacting nodes and local resources.</td>
+      </tr>
+    </tbody>
+  </table>
+
+  </Collapser>
+
+  <Collapser
+    id="OtelComponents-types-usage"
+    title="OpenTelemetry Component types and usage"
+  >
+
+  The OpenTelemetry Collector employs various component types to process and transform the collected telemetry data. These components are essential for enriching, filtering, and formatting the data before it is sent to New Relic.
+
+
```

Comment:
Can we add a sentence similar to?
I want to mention that the info from the able is useful for adding your own config

> You can take advantage of them if you need to add your own config by creating OTel pipelines in the extra_config section.